packages/frontend/src/pages/MinimalSavedExplorer.tsx

```diff
@@ -56,11 +52,13 @@ const MinimalExplorer: FC = () => {
             colorPalette={savedChart.colorPalette}
         >
             <MantineProvider inherit theme={themeOverride}>
-                <StyledLightdashVisualization
-                    // get rid of the classNames once you remove analytics providers
-                    className="sentry-block ph-no-capture"
-                    data-testid="visualization"
-                />
+                <Box mih="inherit" h="100%">
+                    <LightdashVisualization
```

Comment:
shouldn't this be styled instead of adding another?

Reply:
Good callout. I waffled a lot about this one. My hesitation is that `LightdashVisualization` actually returns any one of the visualization types and they all handle style props a little differently. So changing those seems a little risky. Especially since style-components was relying on className and we'd be switching to passing a style. 

So I did this since it is a very specific case: minimal explore view. I didn't want to break visualizaitons in some edge-casey way just to fix this specific view. Am I being too cautious? 😄 If you guys would like, I can dig into this more.

Reply:
I'm gonna merge the less risky version of this. This one the the Box. I tested all of the chart types in minimal and it looks good. If either of you would like, I can follow up and try to clean up the style handling of the various vizes